vaadin-platform-test/src/main/java/com/vaadin/platform/test/ComponentsView.java

```diff
@@ -595,6 +595,8 @@ protected void setPresentationValue(String newPresentationValue) {
         MessageInput messageInput = new MessageInput();
         messageInput.addSubmitListener(e -> log.log(e.getValue()));
 
+        com.vaadin.flow.component.map.Map mapComponent = new com.vaadin.flow.component.map.Map();
```

Comment:
We should not have smoke tests for experimental components in the platform until they go to production